### PR TITLE
Set exec unit type to eks for ts-eks test

### PR DESCRIPTION
**Pull request by gordon-klotho**

### CODE REVIEW

File: ts-eks/test/klotho.yaml [hunk 1]
```diff
@@ -4,63 +4,4 @@ path: .
...
-            type: elasticache
-            pulumi_params_by_type:
-                elasticache:
-                    nodeType: cache.t3.micro
-                    numCacheNodes: 1
-    pubsub:
-        type: sns
-execution_units:
-    main:
         type: eks
```
Comment (jhsinger-klotho) [inline ts-eks/test/klotho.yaml:7]: if this is for pro, dont we want multiple exec units?

Comment (gordon-klotho) [reply ts-eks/test/klotho.yaml:7]: Yeah, this is changing the default so it should apply to all

Comment (jhsinger-klotho) [reply ts-eks/test/klotho.yaml:7]: ah i see, will approve for now, but we want to test both fargate and node types as well as other configurations, so well likely want to refactor this in the future to mirror what we had in the config before the pro split

**Review by yuval-klotho — APPROVED** [review]

File: ts-eks/test/klotho.yaml (hunk 1 above)
Comment (yuval-klotho) [reply ts-eks/test/klotho.yaml:7]: Yeah, I think what we'll _really_ want is the ability to have multiple config files, each turning into its own pair of tests.

**Pull request merged**